mappings/net/minecraft/entity/mob/SpiderEntity.mapping

```diff
@@ -2,6 +2,7 @@ CLASS net/minecraft/class_1628 net/minecraft/entity/mob/SpiderEntity
 	FIELD field_7403 SPIDER_FLAGS Lnet/minecraft/class_2940;
 	METHOD method_26923 createSpiderAttributes ()Lnet/minecraft/class_5132$class_5133;
 	METHOD method_7166 setCanClimb (Z)V
+		ARG 1 canClimb
 	METHOD method_7167 getCanClimb ()Z
```

Comment:
Not part of your PR, but can you rename this method to `canClimb` please?

Reply:
I think that would conflict with an existing method

Reply:
It could still do with a rename. i.e. what's the difference between spider `canClimb` and entity `canClimb`

Reply:
Entity one is for climbing ladders and vines, while the spider one is for climbing on any block

Reply:
Turns out this is actually for when the spider *is* climbing a wall, rather than when it *can* climb a wall.